woocommerce/woocommerce
# Remove references to Sofort

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #40748 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Set up a site using JN Special Ops - include WooCommerce Live Branches with this branch `fix/deprecate-sofort` selected. Do not include WooPayments. Make sure to Opt In to data sharing.
2. Select `United States` as the country. Go to the site, complete to WooCommerce Core onboarding. Make sure to NOT install WooPayments when prompted to (you'll need to untick it on the form in the "Recommended Plugins" section).
3.  On the `WP Admin` dashboard, you should now see the `Payments` option on the menu. Click on it to be taken to the WooPayments setup screen:

<img width="1205" alt="Screenshot 2023-10-13 at 11 06 25" src="https://github.com/woocommerce/woocommerce/assets/11770181/04c4845c-e9dc-442c-a207-9d1f6968cf11">

4. Verify that Sofort is not mentioned, and not among the Payment Icons on this screen.

### Notes

* I also removed Sofort from the Payments Advert Banner which will display when WooPayments is disabled. However, this currently doesn't seem to display - I looked at the instructions in https://github.com/woocommerce/woocommerce/pull/39596 but the banner didn't show. Regardless, checking the code should show that the icon was removed here.
* There are some places Sofort will still display when WooPayments is installed. This will be handled by a separate PR in the WooPayments client ahead of the next plugin release.


</details>
